app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt

```diff
@@ -0,0 +1,12 @@
+package com.thefuntasty.androidprojecttemplate
+
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+
+class TestService : Service() {
+
+
+    //TODO: Test Service only to check new lint checks, but tpyos (typos) shouldnt be a reported problem
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/b0765a4c13fb0e70984828239cb03f4d8d6ea3cf/app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt#L10"></span>This comment contains text that has been defined as forbidden in detekt.</td>
    </tr>
  </tbody>
</table>